parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/BrokerPollSystem.java

```diff
@@ -77,6 +83,13 @@ public BrokerPollSystem(ConsumerManager<K, V> consumerMgr, WorkManager<K, V> wm,
                 committer = Optional.of(consumerCommitter);
             }
         }
+        initMetrics();
+    }
+
+    private void initMetrics() {
+        statusGauge = PCMetrics.getInstance().gaugeFromMetricDef(PCMetricsDef.PC_STATUS, this, poller -> poller.runState.ordinal(), Tag.of("status", runState.name()));
```

Comment:
Wouldn't this create a separate metric per status? I am uncertain if this is ideal

Reply:
Hmm, you are right, will test this and see if it needs to clear previous gauge on status change...